administrator/components/com_associations/views/association/tmpl/edit.php

```diff
@@ -14,15 +14,23 @@
 JHtml::_('formbehavior.chosen', 'select');
 
 $this->app->getDocument()->addScriptDeclaration("
-	function iframeRef( frameRef ) {
-	return frameRef.contentWindow
-		? frameRef.contentWindow.document
-		: frameRef.contentDocument
-	}
+	function triggerSave(frame) {
+		if (frame == 'reference') {
+			window.frames[0].Joomla.submitbutton('" . $this->associatedView . ".apply');
+		}
+		if (frame == 'target') {
+			window.frames[1].Joomla.submitbutton('" . $this->associatedView . ".apply');
```

Comment:
we never know if custom 3º party admin modules add more frames to the window.
the more save option is to get the iframe object by unique id, not by frames array fixed key